tests/test_pyrflx.py

```diff
@@ -314,15 +314,24 @@ def test_accessible_fields_fields_complex(tlv: MessageValue) -> None:
 
 
 def test_valid_message(tlv: MessageValue) -> None:
+    # pylint: disable=pointless-statement
     assert not tlv.valid_message
+    with pytest.raises(RuntimeError, match="cannot create bytestring of invalid message"):
+        tlv.bytestring
```

Comment:
The exact same raises-test is used four times. Introducing a function for this purpose (e.g. `assert_bytestring_error`) would simplify the code. The pylint check should be disabled only for the concrete statement, and not for the whole function.

Reply:
Fixed in https://github.com/Componolit/RecordFlux/pull/431/commits/3cf4e6712922e9de0b6e9b0eae42b6cf36bc0f3d.